README.rst

```diff
@@ -2131,6 +2131,19 @@ Then, add the following command to your CI:
 
 Where ``$COMMIT_RANGE`` is your range of commits, e.g. ``$TRAVIS_COMMIT_RANGE`` or ``HEAD~..HEAD``.
 
+Additionally, you might configure some caching to speed up your continuous builds, e.g. for GitHub Actions, this can be achieved using:
+
+.. code-block:: yaml
+
+      - name: "Cache results of PHP CS Fixer"
+        uses: actions/cache@v1
+        with:
+          path: /tmp/php-cs/
+          key: php-${{ matrix.php-version }}-php-cs-fixer-${{ github.sha }}
```

Comment:
does putting `sha` hear mean each commit will get individual cache storage?

Reply:
please see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4938#issuecomment-625444583 for a in depth description